plugin/src/helpers/files.ts

```diff
@@ -331,13 +331,21 @@ const baseServerReplacements: Array<[string, string]> = [
 
```

Comment:
This is fixing a bug that I found while building this. The change is included in a separate PR: #1480